src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/Release/Release.js

```diff
@@ -92,34 +112,33 @@ export const Release = memo(function Release(props) {
       <CardContent className={styles.CardContent}>
         <WithLoader condition={!loading}>
           {active ? (
-            <section className={styles.ReleaseWrap}>
-              <FieldTypeDropDown
-                name="release"
-                label="Queue item for release"
-                className={styles.SelectRelease}
-                value={selectedRelease}
-                onChange={setSelectedRelease}
-                options={releases.map((release) => {
-                  return {
-                    value: release.ZUID,
-                    component: (
-                      <span>
-                        <span onClick={(evt) => evt.stopPropagation()}>
-                          <AppLink to={`/release/${release.ZUID}`}>
-                            <FontAwesomeIcon icon={faEdit} />
-                          </AppLink>
-                        </span>
-                        &nbsp;{release.name}
-                      </span>
-                    ),
-                  };
-                })}
-              />
+            <section>
+              <FormControl fullWidth>
+                <FormLabel>Queue item for release</FormLabel>
+                <VirtualizedAutocomplete
+                  name="release"
+                  value={selectedRelease}
+                  onChange={(_, option) => {
+                    delete option.component;
+                    setSelectedRelease(option);
+                  }}
+                  placeholder="Select release..."
+                  options={releaseOptions}
+                  startAdornment={
+                    selectedRelease && (
+                      <AppLink to={`/release/${selectedRelease.ZUID}`}>
+                        <FontAwesomeIcon icon={faEdit} />
+                      </AppLink>
+                    )
+                  }
+                />
+              </FormControl>
               <Button
+                fullWidth
                 variant="contained"
                 className={styles.Add}
                 onClick={onAdd}
-                disabled={selectedRelease === "0" || addingMember}
+                disabled={!selectedRelease || addingMember}
```

Comment:
selectedRelease used to default to 0 so this button was disabled until a release is selected. Since now selectedRelease is an object it defaults to null and does a null check until a release is selected